### ISSUE TITLE
I have problem with tutorial

### ISSUE THREAD

**Issue by cminverse** [clojure-android/lein-droid#134]
Here is my [repository](https://github.com/cminverse/lein-droid-tutorial.git) following the lein-droid tutorial:
I made three steps:
1 **`lein new droid superapp my.company.superapp :activity MyActivity :target-sdk 15 :app-name SuperApp`**
2 modify the version-code in **AndroidManifest.template.xml**
3 then **`lein droid doall`** come to this:
```
	......
Compiling my.company.superapp.main
Reflection warning, neko/activity.clj:109:29 - reference to field getName can't be resolved.
Reflection warning, my/company/superapp/main.clj:19:25 - reference to field getText can't be resolved.
Exception in thread "main" java.lang.NullPointerException, compiling:(/tmp/form-init3398839089481863393.clj:1:73)
	at clojure.lang.Compiler.load(Compiler.java:7249)
	at clojure.lang.Compiler.loadFile(Compiler.java:7175)
	at clojure.main$load_script.invoke(main.clj:280)
	......
	at clojure.main.main(main.java:37)
Caused by: java.lang.NullPointerException
	at clojure.lang.Reflector.invokeNoArgInstanceMember(Reflector.java:301)
	......
	Compilation failed.
```
**lein --version** `Leiningen 2.5.2 on Java 1.8.0_51 Java HotSpot(TM) 64-Bit Server VM`
**BUT** here is [**another repository**](https://github.com/cminverse/cljondroid.git) in old version which worked well for me. I noticed that the **neko/neko** is **"3.2.0"** instead of **"4.0.0-alpha4"**
I'm a newbie, got confused by this tutorial problem. Is this a bug??

**Comment by alexander-yakushev** [comment]
Indeed it was a bug in Neko, sorry for that. Please try `0.4.0-alpha5` and say if that fixes the issue.

**Comment by cminverse** [comment]
Yes, it fixed in ``neko "0.4.0-alpha5"``.

**Comment by alexander-yakushev** [comment]
Now fixed in `lein droid 0.4.2` too, the template contains correct Neko dependency.
